glowing_waffles/notebooks/observe class.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<p>I was a little confused by the output here -- had you re-run this a few times?</p>

 _Reply via <a href='https://app.reviewnb.com/glowing-waffle/glowing-waffles/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>